libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -144,6 +144,7 @@ @implementation SFOAuthCoordinator
 @synthesize oauthCoordinatorFlow        = _oauthCoordinatorFlow;
 @synthesize userAgentForAuth            = _userAgentForAuth;
 @synthesize origWebUserAgent            = _origWebUserAgent;
+@synthesize brandLoginPath              = _brandLoginPath;
```

Comment:
@wmathurin I will remove it. When I started I had a custom setter. Doesn't need an explicit declaration.